README.md

```diff
@@ -1,68 +1,26 @@
-# Google API Client [![Documentation](https://img.shields.io/badge/docs-Google%3A%3AApis-red.svg)](https://googleapis.dev/ruby/google-api-client/latest/Google/Apis.html) [![Gem Version](https://badge.fury.io/rb/google-api-client.svg)](https://badge.fury.io/rb/google-api-client)
+# Legacy REST Clients for Google APIs
```

Comment:
Is it required to label all of these clients as "legacy"? I find it a bit confusing in those cases where no newer client exists.

Reply:
I've been going back and forth on that. But you're right, it's probably premature to call them "legacy" at this point. I changed the label to "simple".